utils/uart_utils.h

```diff
@@ -10,4 +10,4 @@
 
 int uart_init();
 void uart_write(const int fd, char *cmd);
-char *uart_read(const int fd);
\ No newline at end of file
+char *uart_read(const int fd);
```

Comment:
These operations should be encapsulated in HAL.

Reply:
New issues #32 for this.